app/src/main/java/com/example/drawingactivity/BitmapFromUri.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.drawingactivity
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.Drawable
+import android.net.Uri
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.CustomTarget
+import com.bumptech.glide.request.transition.Transition
+
+/**
+ * This Class is inherits the Glide Class Custom Target and allows you to set it to a bitmap.
+ * onResourceReady executes when the load operation is done.
+ */
+class BitmapFromUri(val onComplete: (Bitmap) -> Unit): CustomTarget<Bitmap>() {
+
+    override fun onResourceReady(resource: Bitmap, transition: Transition<in Bitmap>?) {
+        onComplete(resource)
+    }
+
+    override fun onLoadCleared(placeholder: Drawable?) {}
+}
+
+/**
+ * Shorthand Class to insert a bitmap into a location that accepts a bitmap using Glide.
+ */
+class BitmapFromGallery(context: Context, uri: Uri,val onComplete: (Bitmap) -> Unit) {
```

Comment:
This needs an error lambda for what to do on error

Reply:
Added an error lambda and use an error bitmap when something goes wrong